### ISSUE TITLE
NullPointerException - IntelliJ 2017.1 EAP

### ISSUE THREAD

**Issue by CarsonF** [erikzielke/GoToProject#6]
~Randomly throughout the day IntelliJ crashes. OSX gives the "app stopped unexpectedly...." error popup.~

```
IntelliJ IDEA 2017.1 EAP
Build #IU-171.3691.12, built on March 1, 2017
IntelliJ IDEA EAP User
Expiration date: March 31, 2017
JRE: 1.8.0_112-release-724-b5 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.11.6
```

~Disabling this plugin stops this.~ The only thing I see in the logs is a `NullPointerException`.

```
java.lang.NullPointerException
	at org.github.erikzielke.gotoproject.GoToProjectProjectComponent.projectClosed(GoToProjectProjectComponent.java:42)
	at com.intellij.openapi.project.impl.ProjectImpl.a(ProjectImpl.java:405)
	at com.intellij.openapi.project.impl.ProjectImpl.access$200(ProjectImpl.java:69)
	at com.intellij.openapi.project.impl.ProjectImpl$MyProjectManagerListener.projectClosed(ProjectImpl.java:434)
	at com.intellij.openapi.project.impl.ProjectManagerImpl$1.projectClosed(ProjectManagerImpl.java:105)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.i(ProjectManagerImpl.java:701)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:605)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:995)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.closeProject(ProjectManagerImpl.java:602)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.closeAndDispose(ProjectManagerImpl.java:626)
	at com.intellij.ide.impl.ProjectUtil.closeAndDispose(ProjectUtil.java:89)
	at com.intellij.openapi.wm.impl.IdeFrameImpl$4.windowClosing(IdeFrameImpl.java:262)
...
```

**Comment by CarsonF** [comment]
Ok it just crashed with this plugin disabled, so this is not the cause of that. Still the null pointer though ;)

**Comment by erikzielke** [comment]
Okay :)   I hope I will get team to fix the NPE soon

**Comment by CarsonF** [comment]
Fixed with bcd427c92499108dc123175e605ba15e70344d36
